README.md

```diff
@@ -30,5 +30,296 @@
 
 # Triton Inference Server
 
-**NOTE: You are currently on the r21.03 branch which tracks stabilization
-towards the next release. This branch is not usable during stabilization.**
+Triton Inference Server provides a cloud and edge inferencing solution
+optimized for both CPUs and GPUs. Triton supports an HTTP/REST and
+GRPC protocol that allows remote clients to request inferencing for
+any model being managed by the server. For edge deployments, Triton is
+available as a shared library with a C API that allows the full
+functionality of Triton to be included directly in an
+application.
+
+## What's New in 2.8.0
+
+* The Windows build now supports ONNX models.
+
+* Repository agent is a new extensibility C API added to Triton that
+  allows implementation of custom authentication, decryption,
+  conversion, or similar operations when a model is loaded. See
+  https://github.com/triton-inference-server/server/blob/master/docs/repository_agents.md
+
+* An OpenVINO backend is added to Triton to enable the execution of
+  OpenVINO models on CPUs.
+  https://github.com/triton-inference-server/openvino_backend
+
+* The PyTorch backend is now maintained in its own repository:
+  https://github.com/triton-inference-server/pytorch_backend
+
+* The ONNX Runtime backend is now maintained in its own repository:
+  https://github.com/triton-inference-server/onnxruntime_backend
+
+* The Jetson release of Triton now supports the shared-memory protocol
```

Comment:
Can we clarify that this is "system" shared memory? If the users try to use cuda shared memory, it will not work

Reply:
Fixed